packages/csv-parser-price/my-prices.csv

```diff
@@ -0,0 +1,4 @@
+variant-sku,value.currencyCode,value.centAmount,customType,customField.foo
```

Comment:
I think we should group this in a folder called 'samples' or something, not just putting it in the root